shared/js/media/remote_controls.js

```diff
@@ -368,14 +338,14 @@ MediaRemoteControls.prototype.notifyAppInfo = function(info) {
  * @param {Object} metadata
  */
 MediaRemoteControls.prototype.notifyMetadataChanged = function(metadata) {
+  var onsuccess = function() {
+    // To do nothing here, expect onsuccess normally.
+  };
+  var onerror = function() {
+    console.log('Sending Metadata error');
+  };
   // Send the new metadata via bluetooth.
-  if (this.defaultAdapter) {
-    var request = this.defaultAdapter.sendMediaMetaData(metadata);
-
-    request.onerror = function() {
-      console.log('Sending Metadata error');
-    };
-  }
+  this._bluetoothHelper.sendMediaMetaData(metadata, onsuccess, onerror);
```

Comment:
Also, for the readability and keeping the same code style, let's move back this method call before declaring the onsuccess and onerror functions.